livekit-agents/livekit/agents/worker.py

```diff
@@ -158,6 +158,11 @@ class WorkerOptions:
 
     Defaults to 0.75 on "production" mode, and is disabled in "development" mode.
     """
+    max_job_memory_usage: float = 0
```

Comment:
Let's put something like 300MB by default?

Reply:
I would prefer to leave it empty and let the users to decide.

This might be a breaking change if there is a default limit. I have tested that the minimal assistant agent the RAM usage is around 150M, but it could be higher if users load some other data or publishing/receiving videos. Those agents will be killed unexpectedly after upgrading.

Reply:
it makes sense to not break existing apps. Is it possible to have a "warn" mode where we initially let folks know about the high memory usage without killing the requests? Since most users don't pay attention to memory, making memory issues clearly visible would be beneficial.

maybe we should have two separate options?
 - `job_memory_warn`
 - `job_memory_limit`

Reply:
`job_memory_warn_mb` (defaults to 300) and `job_memory_limit_mb` are added.